app/system/ServiceProvider.php

```diff
@@ -310,7 +310,10 @@ protected function setConfiguration()
 
         $this->app->resolving('translator.localization', function ($localization, $app) {
             $app['config']->set('localization.locale', setting('default_language', $app['config']['app.locale']));
-            $app['config']->set('localization.supportedLocales', setting('supported_languages', []));
+            $supportedLocales = setting('supported_languages', []);
+            if (!$supportedLocales OR !count($supportedLocales))
+                $supportedLocales = ['en'];
+            $app['config']->set('localization.supportedLocales', $supportedLocales);
```

Comment:
Maybe use a ternary operator instead? Empty arrays evaluates to false.

Reply:
Yep much neater - thanks

Reply:
Yup but `??` is a null coalescing operator. Change to `?:`  ;)

Reply:
Done :)